packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -49,6 +49,11 @@
     // Allow local storage.
     webView.getSettings().setDomStorageEnabled(true);
 
+    // Allow zoom and hide the zoom in and zoom out buttons.
+    webView.getSettings().setSupportZoom(true);
```

Comment:
Should this setting be controllable by the app developer? any I'd suggest splitting these 2 features to separate PRs